docs/metrics/latest

```diff
@@ -533,6 +533,21 @@ etcd_server_is_leader
 # type: "counter"
 etcd_server_leader_changes_seen_total
 
+# name: "etcd_server_is_learner"
+# description: "Whether or not this member is a learner. 1 if is, 0 otherwise."
+# type: "gauge"
+etcd_server_is_learner
+
+# name: "etcd_server_learner_promote_failures"
+# description: "The total number of learner promote  failures (likely learner not ready) while this member is leader."
```

Comment:
Found a double space in this line, between "promote" and "failures".

Maybe change the wording to match the other metric, something like "The total number of failed learner promotions ..."